README.md

```diff
@@ -9,6 +9,7 @@ This cookbook provides resources for configuring and managing nodes running in A
 - CloudWatch Instance Monitoring (`instance_monitoring`)
 - DynamoDB (`dynamodb_table`)
 - EBS Volumes (`ebs_volume`)
+- EC2 Instance DisableApiTermination (`instance_term_protection`)
```

Comment:
The title of the resource seems a bit confusing here compared to the simple description below